app/src/main/java/com/babylon/wallet/android/presentation/ui/composables/SimpleAccountCard.kt

```diff
@@ -68,7 +72,8 @@ fun SimpleAccountCard(
 @Composable
 fun SimpleAccountCard(
     modifier: Modifier = Modifier,
-    account: Network.Account
+    account: Network.Account,
+    showName: Boolean = true
```

Comment:
"shouldShowName" but the most important.... is this parameter needed? 
Everywhere is set to true. Only in one preview is set as false.

Reply:
initial figma designs had no name, this is why I added it. But Matt requested to use unnamed. Will remove